modules/itest/itest-core/src/test/java/com/enonic/xp/core/scheduler/SchedulerServiceImplTest.java

```diff
@@ -426,8 +426,8 @@ void get()
 
         assertEquals( name, scheduledJob.getName() );
         assertEquals( descriptor, scheduledJob.getDescriptor() );
-        assertEquals( calendar.getCronValue(), ( (CronCalendar) scheduledJob.getCalendar() ).getCronValue() );
-        assertEquals( calendar.getTimeZone(), ( (CronCalendar) scheduledJob.getCalendar() ).getTimeZone() );
+        assertEquals( calendar.getCronValue(), ( (CronCalendarImpl) scheduledJob.getCalendar() ).getCronValue() );
```

Comment:
why would one need to cast?

Reply:
ScheduledJob.getCalendar() returns ScheduleCalendar which is too common to fetch fields.